<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KlasfotoSettings</class>
 <widget class="QDialog" name="KlasfotoSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>721</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tblSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>451</width>
     <height>241</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <column>
    <property name="text">
     <string>Class</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Graad</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Grade</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>None</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>660</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>660</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Confrim</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAllGraad">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Graad</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAllGrade">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Grade</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAllNone">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ALL:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>441</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Sample</string>
   </property>
   <widget class="QLabel" name="imgPreview">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Font colour</string>
   </property>
   <widget class="QRadioButton" name="rbtWhite">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>White</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtBlack">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Black</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
